src/main/java/uk/gov/justice/services/test/domain/EventsChecker.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.services.test.domain;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static uk.gov.justice.services.test.DomainTest.eventNameFrom;
+import static uk.gov.justice.services.test.DomainTest.generatedEventAsJsonNode;
+import static uk.gov.justice.services.test.DomainTest.jsonNodeWithoutMetadataFrom;
+
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+
+public class EventsChecker {
+
+    public static void compareEvents(List<JsonNode> expectedEvents, List<Object> actualEvents) {
+
+        assertThat(createMessage(expectedEvents, actualEvents, null), expectedEvents.size(), is(actualEvents.size()));
+
+        int index = 0;
+        for (JsonNode jsonNode : expectedEvents) {
+
+            final Object generatedEvent = actualEvents.get(index);
```

Comment:
Why wouldn't you rename this to be consistent with the use of "actual" instead of "generated"?